src/queries/voting-data.ts

```diff
@@ -11,7 +11,7 @@ export interface VotingData {
 }
 
 const aragonEnv = {
-  network: parseInt(process.env.CHAIN_ID, 10),
+  network: parseInt(process.env.CHAIN_ID as string, 10),
```

Comment:
Environment variables are always read as strings, so this is redundant. The issue you are probably trying to solve with this cast is that the variable is possibly `undefined`. 

Either account for that possibilty (like in line 15 below) or assert that you KNOW it will never be undefined with `process.env.CHAIN_ID!`

Reply:
I opted for the process.env.CHAIN_ID! as we are putting the number in the example.env